libraries/AP_NavEKF3/AP_NavEKF3_core.cpp

```diff
@@ -1873,6 +1873,22 @@ void NavEKF3_core::ForceSymmetry()
 // if states are inactive, zero the corresponding off-diagonals
 void NavEKF3_core::ConstrainVariances()
 {
+    // Covariance constraints as of March 2025
```

Comment:
I suggest that we modify the log messages to contain this data.  It will be much more useful in the log analysis tools than a comment.

Reply:
@peterbarker Do you mean adding this data in brackets to the log message documentation?

Reply:
> @peterbarker Do you mean adding this data in brackets to the log message documentation?

Yes, renaming the existing V00 fields to something more descriptive and adding all of the unit metadata etc etc.

Not to say we can't merge this as-is, but having all of this in the tools without having to go look at the code would be nice.